app/api/assistants/threads/[threadId]/cancel-runs/route.ts

```
@@ -0,0 +1,48 @@
+    }
+
+    return NextResponse.json({
+      message: `Cancelled ${cancelledCount} active runs${failedCount > 0 ? `, failed to cancel ${failedCount} runs` : ''}`,
+      cancelledCount,
+      failedCount
+    });
+
+    return NextResponse.json({
+      message: `Cancelled ${cancelledCount} active runs`,
+      cancelledCount
+    });
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)